variables.env

```diff
@@ -7,7 +7,8 @@
 # Append -prerelease while working on releases.
 # Remove -prerelease when ready to merge release to the release branch.
 # Call it main on the main branch.
-WMDE_RELEASE_VERSION=main
+WMDE_RELEASE_VERSION=2.3.0
```

Comment:
I'd remove this variable completely in order to stress that there is not central version number anymore.

Reply:
It's used as the wikibase version. I can rename the variable to reflect that.